CriteoPublisherSdk/Sources/Network/Serializers/CR_BidRequestSerializer.m

```diff
@@ -156,11 +156,21 @@ - (NSArray *)slotsWithCdbRequest:(CR_CdbRequest *)cdbRequest {
     } else if (adUnit.adUnitType == CRAdUnitTypeRewarded) {
       slotDict[CR_ApiQueryKeys.bidSlotsIsRewarded] = @(YES);
     }
+    if (config.isMRAIDEnabled) {
```

Comment:
do you mind adding extra condition like
(adUnit.adUnitType==CRAdUnitTypeBanner || adUnit.adUnitType==CRAdUnitTypeInterstitial)

Reply:
ok, sure